README.md

```diff
@@ -68,6 +70,31 @@ example_value_count{environment="beta",id="id-A"} 1
 example_value_count{environment="beta",id="id-C"} 3
 ```
 
+Headers & Additional Endpoints
+=============
+An example configuration for multiple endpoints, with headers:
+
+```
+- endpoint: http://example.com/endpoint-1
+  headers:
+  - Content-Type: application/json
```

Comment:
I think this YAML causes parsing error as [`ModuleConfig`](https://github.com/kawamuray/prometheus-json-exporter/pull/4/files#diff-719916233fcf8fcf593dcaef527bb211R11) declares `Headers` as `map[string]string` but these lines are interpreted as list of strings.
AFAIK multiple headers with same name is allowed in HTTP but considering practical use case I think it's okay to get rid of leading dash `-` and make these "map".